### PR TITLE
Add orga slides dec 2

### PR DESCRIPTION
**Pull request by uekerman**
@ajaust Did I forget anything?

### CODE REVIEW

File: organization/material/organizational_remarks_week7_slides.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+---
+
+## Challenge
+
+- Information on **second step** on [GitLab](https://gitlab-sim.informatik.uni-stuttgart.de/simulation-software-engineering/challenge/-/issues/3)
+    - Now, the *"other half"* presents.
+    - Still, everybody needs to hand in a report (deadline: Dec 16, 15:45).
+- Please open an issue in the [challenge repo](https://gitlab-sim.informatik.uni-stuttgart.de/simulation-software-engineering/challenge/) explaining what you want to work on for step 3. We will then use these issues to further discuss the progress of the contribution. Provide links to issues, pull requests etc.
+    - Deadline: Jan 10, 23:59 (to open the issue)
```
Comment (ajaust) [inline organization/material/organizational_remarks_week7_slides.md:63]:
```suggestion
    - Deadline (for opening the issue): Jan 10 2022, 23:59
```

File: organization/material/organizational_remarks_week7_slides.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    color: orange;
+  }
+  .reveal section img {
+    background:none;
+    border:none;
+    box-shadow:none;
+  }
+</style>
+
+# Organizational Things
```
Comment (ajaust) [inline organization/material/organizational_remarks_week7_slides.md:36]: Should we start numbering these headlines or give them a date? 

Something like
```markdown
# Organizational Things 2021-12-02
```
or

```markdown
# Organizational Things (3)
```

**Review by ajaust — COMMENTED** [review]
The looks overall good to me. I only have tiny remarks which could also be ignored. :smile:

File: organization/material/organizational_remarks_week7_slides.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+---
+
+## Poll on First 5 Weeks
+
+- Thanks for participating!
+- Many wrote that they liked the concept. Made us very happy. :blush:
+- **Multiple sources of content (website, GitLab, GitHub)**
+    - We are aware of the problem and are thinking about solutions.
+    - Reason 1: you should learn GitLab + GitHub
+    - Reason 2: private information (GitLab) vs. public information (GitHub)
```
[the remark this answers is not in the document]
Comment (uekerman) [reply organization/material/organizational_remarks_week7_slides.md:47]: :+1: I'll mention it

**Pull request merged**